### PR TITLE
Mentee application

### PR DESCRIPTION
**Pull request by Nicanor008**
## What's this PR about
Allows a mentee to send an application to join the Shoman platform.

## How to test this PR
- clone and open this project
- checkout to this branch `git checkout mentee-apply-#24`
- run development env `gatsby develop`, of course, you should have gatsby installed globally. if not, check Readme.md documentation.
- on the browser, open `http://localhost:4000/apply`

## Issue
#24 

## Output Screenshots
Screenshot 2020-12-09 at 19 49 38

### CODE REVIEW

File: src/components/auth/apply/applyForm.js [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+                    onChange={onSelectChange}
+                    name="track"
+                  >
+                    <MenuItem value="Frontend track">Frontend</MenuItem>
+                    <MenuItem value="Backend track">Backend</MenuItem>
+                    <MenuItem value="FullStack track">FullStack</MenuItem>
+                    <MenuItem value="Mobile track">Mobile Dev</MenuItem>
+                    <MenuItem value="track">
+                      {" "}
+                      {/* solves bug from material ui */}
```
Comment (Nyakaru) [inline src/components/auth/apply/applyForm.js:187]: What bug is this?

File: src/components/auth/signup/signup.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.signupImage {
+    position: sticky;
+    height: 100%;
+    align-self: center;
+}
+@media (max-width: 768px) {
+    .signupImage {
+        display: none;
+    }
+}
```
Comment (Nyakaru) [inline src/components/auth/signup/signup.scss:10]: Ditto

File: src/components/input.js [hunk 3]
```diff
@@ -43,3 +43,20 @@ const InputComponents = (props) => {
 }
 
 export default InputComponents
+
+// export const SelectInput = props => {
```
Comment (Nyakaru) [inline src/components/input.js:47]: You still need this function?

File: src/components/auth/apply/applyForm.js (hunk 1 above)
Comment (Nicanor008) [reply src/components/auth/apply/applyForm.js:187]: Not the best solution for now, will research more in to it later
Screenshot 2020-12-10 at 09 39 43

**Review by Nyakaru — APPROVED** [review]
Please merge this so that I can take over with auth stuff

**Pull request merged**